content/adventures/en.yaml

```diff
@@ -1571,7 +1571,28 @@ adventures:
                             turn right
                     ```
                 start_code: "# place your code here"
-
+            11:
```

Comment:
Lovely adventure!! let me know when this is ready to fully review, but this looks great!!